src/vue-cookie.js

```diff
@@ -24,6 +24,10 @@
             return Cookie.get(name);
         },
 
+        getAll: function () {
```

Comment:
Just a wrapper for `tiny-cookie`'s `getAll()` method. Also updated to the latest version of `tiny-cookie`.